### ISSUE TITLE
How to run my cart? Getting error.

### ISSUE THREAD

**Issue by greysonp** [picolove/picolove#54]
Hey there!

I'm new to Lua and Love stuff. I've installed Love and have successfully run a "hello, world."  How do I use picolove to run my pico-8 cart?

Currently, if I just run `love .` in the picolove directory, I get this screen:
![picolove error](https://cloud.githubusercontent.com/assets/1358731/16491549/47dbbb4e-3e93-11e6-8f62-813189818b65.png)

Any help is appreciated, thanks!

**Comment by greysonp** [comment]
Ok, so I confirmed that picolove is using a version of the love.graphics.newMesh() function that was removed in 10.0 (see [documentation](https://love2d.org/wiki/love.graphics.newMesh)).

I also see that this doesn't seem to be maintained anymore, and so I've switched to [this fork](https://github.com/gamax92/picolove), which seems to have fixed all of these issues.

Thanks so much for starting this!
